scene/main/scene_tree.h

```diff
@@ -236,7 +236,9 @@ class SceneTree : public MainLoop {
 		GROUP_CALL_UNIQUE = 4,
 	};
 
-	_FORCE_INLINE_ Window *get_root() const { return root; }
+	_FORCE_INLINE_ Window *get_root() const {
+		return root;
+	}
```

Comment:
This is actually a bug in clang-format 14, it shouldn't do that. https://github.com/llvm/llvm-project/issues/54901
It's fixed but only in the unreleased clang-format 15 :(

Reply:
You might be able to drop this commit and still fool clang-format 14 btw, so far I've managed to not commit these bogus changes from that version.

Reply:
I can run `git commit` with `--no-verify`.

Reply:
I've dropped the commit and static checks have passed.